src/Conversion/ONNXToTOSA/ONNXToTOSALegalizeUtils.cpp

```diff
@@ -48,7 +48,7 @@ Value buildOnnxToTosaPaddingConstOp(mlir::PatternRewriter &rewriter,
 
   // Create a new pad vec in the right format
   // ONNX : [b1, b2, b3, b4, e1, e2, e3, e4]
-  // TOSA :[[b1, e1], [b2, e2], [b3, e3], [b4, e4]]
+  // TOSA :[b1, e1, b2, e2, b3, e3, b4, e4]
```

Comment:
@jorickert This update was made due to the following error message :

`/Users/christophermunoz/repos/cmunoz1/onnx-mlir/test/mlir/conversion/onnx_to_tosa/NN/AveragePool.mlir:139:8: error: 'tosa.pad' op operand #1 must be 1D tensor of 32-bit signless integer or 64-bit signless integer values, but got 'tensor<4x2xi64>'`

Just want to make sure this update is fine.... I am not too familiar with TOSA, so asking just in case.

Reply:
LLVM made some updates to TOSA as shown in this PR: https://github.com/llvm/llvm-project/pull/122547